winterfell/utils/constants.js

```diff
@@ -22,3 +22,12 @@ module.exports.biscuitKeys = {
   idmeClientId: 'id-me-client-id',
   idmeSecretId: 'id-me-client-secret'
 };
+
+module.exports.ERROR_CODES = {
```

Comment:
ERROR_CODES constant keys are good for server-side. But I think the message that it is associated with it, should be on the front end.
So I think the server response could send a `eusernametaken` key back.
But on the front end, it is more proper to take that key to get a message.